### PR TITLE
Add Tags for RDS Database Instances

### PR DESCRIPTION
**Pull request by klauern**
We use tagging for a lot of auditing purposes, so having this data returned to us in a ScoutSuite run is very useful.  This PR adds those tags in.

One unfortunate side-effect is that this requires a new API call to [`ListTagsForResources`](https://docs.aws.amazon.com/AmazonRDS/latest/APIReference/API_ListTagsForResource.html)

### CODE REVIEW

**Comment by klauern** [comment]
I'm not sure how I should test this addition.  Can someone point me to an example test that exists for RDS currently that I could add this to?

File: ScoutSuite/__main__.py [hunk 1]
```diff
@@ -113,6 +113,8 @@ def run(provider,
     """
 
     loop = asyncio.get_event_loop()
+    if loop.is_closed():
```
Comment (j4v) [inline ScoutSuite/__main__.py]: Why are you adding this?

Comment (klauern) [reply ScoutSuite/__main__.py]: gah, this is unrelated.  This is so it will run in a Lambda.

Comment (klauern) [reply ScoutSuite/__main__.py]: I removed this.  I'll try to put it in another PR later

**Comment by klauern** [comment]
One note I forgot to mention was that this page will have to add the `rds:ListTagsForResource` permission: https://github.com/nccgroup/ScoutSuite/wiki/AWS-Minimal-Privileges-Policy

**Comment by j4v** [comment]
Looks :100: , merging.

> One note I forgot to mention was that this page will have to add the rds:ListTagsForResource permission: https://github.com/nccgroup/ScoutSuite/wiki/AWS-Minimal-Privileges-Policy

Thanks for mentioning :medal_military:

**Pull request merged**